<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>模糊查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quickSearch">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>12</y>
     <width>461</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="quickSerchButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>快速检索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>快速检索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setQueryData">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>检索</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>144</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="inputData1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="inputData2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="inputData3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="inputData4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="inputData5"/>
    </item>
    <item>
     <widget class="QLineEdit" name="inputData6"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>53</y>
     <width>71</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>检索条件1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>检索条件2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>检索条件3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>检索条件4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>检索条件5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>检索条件6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>151</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QComboBox" name="searchCondition1"/>
    </item>
    <item>
     <widget class="QComboBox" name="searchCondition2"/>
    </item>
    <item>
     <widget class="QComboBox" name="searchCondition3"/>
    </item>
    <item>
     <widget class="QComboBox" name="searchCondition4"/>
    </item>
    <item>
     <widget class="QComboBox" name="searchCondition5"/>
    </item>
    <item>
     <widget class="QComboBox" name="searchCondition6"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>61</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QComboBox" name="compareCond1"/>
    </item>
    <item>
     <widget class="QComboBox" name="compareCond2"/>
    </item>
    <item>
     <widget class="QComboBox" name="compareCond3"/>
    </item>
    <item>
     <widget class="QComboBox" name="compareCond4"/>
    </item>
    <item>
     <widget class="QComboBox" name="compareCond5"/>
    </item>
    <item>
     <widget class="QComboBox" name="compareCond6"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QComboBox" name="fuzzyQuery1"/>
    </item>
    <item>
     <widget class="QComboBox" name="fuzzyQuery2"/>
    </item>
    <item>
     <widget class="QComboBox" name="fuzzyQuery3"/>
    </item>
    <item>
     <widget class="QComboBox" name="fuzzyQuery4"/>
    </item>
    <item>
     <widget class="QComboBox" name="fuzzyQuery5"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
